library/src/main/java/com/yayandroid/locationmanager/helper/LogUtils.java

```diff
@@ -14,24 +17,28 @@ public static void enable(boolean isEnabled) {
         LogUtils.isEnabled = isEnabled;
     }
 
+    public static void setLogger(Logger logger) {
```

Comment:
Can you add `NonNull` annotation please?

Reply:
Sure! I also added it to LocationManager and removed the contradictory test.